### PR TITLE
Adding a new command equivalent to linux shell 'whoami'

### PR DESCRIPTION
**Pull request by SrinivasChilveri**
This command will help user to get the information related to the Org and Space they have targeted and the role associated with them.

***Output :***  
```
$ cf user-info
Getting user information......
User    Org       Space       Role
asif    asifOrg   asifSpace   ORG MANAGER, BILLING MANAGER, SPACE MANAGER, SPACE DEVELOPER
```
Documentation for this command  : https://github.com/asifhuawei/Docs/blob/master/user-info.md

### CODE REVIEW

**Comment by drnic** [comment]
There is also "cf target" - which info is missing from this command output? Perhaps merge the additional info into this existing command?

**Comment by asifhuawei** [comment]
Hi @drnic 
Apart from the information in ```cf target``` this command gives the roles of the Org and Space which user has targeted. At present there is no command which gives the specific information of roles of the current user.

As cf target retrieves the information of the names of Org and Spaces from the local config files and it doesn't queries for these information to CC and the roles information is not present in local config files so it was better option to write the standalone command for this.

But we are open for change so if it needs some changes then we will be happy to make changes to this PR.

**Comment by asifhuawei** [comment]
Hi @goehmen 

Recently we saw two new endpoints [org-user_roles](https://www.pivotaltracker.com/n/projects/966314/stories/88712782) and [space-user_roles](https://www.pivotaltracker.com/n/projects/966314/stories/89467572) to retrieve the user roles, do you want us to use those two new endpoints for implementing this new command.

Waiting for your inputs to make changes to this PR.

File: cf/i18n/resources/fr_FR.all.json [hunk 1]
```diff
@@ -5153,5 +5153,30 @@
       "id": "CF_NAME unshare-private-domain ORG DOMAIN",
       "translation": "CF_NAME unshare-private-domain ORG DOMAIN",
       "modified": false
+   },
+   {
+      "id": "Show user-info with roles",
+      "translation": "Show user-info with roles",
```
Comment (maximilien) [inline cf/i18n/resources/fr_FR.all.json]: Here is the French translation, which I have been maintaining...

```
 "Montrer user-info avec ces roles"
```

File: cf/i18n/resources/fr_FR.all.json [hunk 2]
```diff
@@ -5153,5 +5153,30 @@
...
+      "modified": false
+   },
+   {
+      "id": "CF_NAME user-info",
+      "translation": "CF_NAME user-info",
+      "modified": false
+   },
+   {
+      "id": "Getting user information...",
+      "translation": "Getting user information...",
```
Comment (maximilien) [inline cf/i18n/resources/fr_FR.all.json]: French translation:

```
"En trains d'obtenir information pour l'utilisateur courrant..."
```

File: cf/i18n/resources/fr_FR.all.json [hunk 3]
```diff
@@ -5153,5 +5153,30 @@
...
+      "modified": false
+   },
+   {
+      "id": "User",
+      "translation": "User",
```
Comment (maximilien) [inline cf/i18n/resources/fr_FR.all.json]:
```
"Utilisateur"
```

File: cf/i18n/resources/fr_FR.all.json [hunk 4]
```diff
@@ -5153,5 +5153,30 @@
...
+      "modified": false
+   },
+   {
+      "id": "Role",
+      "translation": "Role",
```
Comment (maximilien) [inline cf/i18n/resources/fr_FR.all.json]:
```
"Rôle"
```

File: cf/commands/user/user-info_test.go [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+			runCommand()
+
+			Expect(ui.Outputs).To(ContainSubstrings(
+				[]string{"Getting user information..."},
+				[]string{"User", "Org", "Space", "Role"},
+				[]string{"user", "Org1", "Space1", "SPACE MANAGER", "SPACE DEVELOPER", "SPACE AUDITOR", "ORG MANAGER", "BILLING MANAGER", "ORG AUDITOR"},
+			))
+		})
+
+		It("should fail when QueryParam is wrong", func() {
```
Comment (maximilien) [inline cf/commands/user/user-info_test.go]: Seems you might be missing a test when the user name is bogus?

File: cf/api/organizations/organizations_test.go [hunk 6]
```diff
@@ -81,6 +81,83 @@ var _ = Describe("Organization Repository", func() {
...
+
+			org, apiErr := repo.GetOrgRoleForUser("billing_manager_guid", "Org1")
+			Expect(handler).To(HaveAllRequestsCalled())
+			Expect(apiErr).NotTo(HaveOccurred())
+			Expect(org.Name).To(Equal(existingOrg.Name))
+
+		})
+
+		It("checking for apiErr", func() {
+
```
Comment (maximilien) [inline cf/api/organizations/organizations_test.go]: get rid of this space, distracting :)

File: cf/api/organizations/organizations_test.go [hunk 7]
```diff
@@ -81,6 +81,83 @@ var _ = Describe("Organization Repository", func() {
...
+			testserver, handler, repo := createOrganizationRepo(req)
+			defer testserver.Close()
+			existingOrg := models.Organization{}
+			existingOrg.Name = "Org1"
+
+			org, apiErr := repo.GetOrgRoleForUser("billing_manager_guid", "Org1")
+			Expect(handler).To(HaveAllRequestsCalled())
+			Expect(apiErr).NotTo(HaveOccurred())
+			Expect(org.Name).To(Equal(existingOrg.Name))
+
```
Comment (maximilien) [inline cf/api/organizations/organizations_test.go]: why this new line?

**Comment by camelpunch** [comment]
@asifhuawei closing due to inactivity. Feel free to reopen, rebase and make the requested fixes.

**Pull request closed without merging**